lightning/src/ln/onchain_utils.rs

```diff
@@ -1,15 +1,48 @@
-//! Utilities for computing witnesses weight and feerate computation for onchain operation
+// This file is Copyright its original authors, visible in version control
+// history.
+//
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
 
+//! Various utilities to assemble claimable outpoints in package of one or more transactions. Those
+//! packages are attached metadatas, guiding their aggregation or fee-bumping re-schedule. This file
+//! also includes witness weight computation and fee computation methods.
+
+use bitcoin::blockdata::transaction::{TxOut,TxIn, Transaction, SigHashType};
+use bitcoin::blockdata::transaction::OutPoint as BitcoinOutPoint;
+use bitcoin::blockdata::script::Script;
+
+use bitcoin::hash_types::Txid;
+
+use bitcoin::secp256k1::key::{SecretKey,PublicKey};
+
+use ln::PaymentPreimage;
+use ln::chan_utils::{TxCreationKeys, HTLCOutputInCommitment};
+use ln::chan_utils;
 use ln::msgs::DecodeError;
+use ln::onchaintx::OnchainTxHandler;
+use chain::keysinterface::Sign;
+use util::byte_utils;
+use util::logger::Logger;
 use util::ser::{Readable, Writer, Writeable};
 
+use std::cmp;
+use std::ops::Deref;
+
+const MAX_ALLOC_SIZE: usize = 64*1024;
+
+
+/// witnessScript markers used to compute the expected witness weight.
 #[derive(PartialEq, Clone, Copy)]
 pub(crate) enum InputDescriptors {
 	RevokedOfferedHTLC,
 	RevokedReceivedHTLC,
 	OfferedHTLC,
 	ReceivedHTLC,
-	RevokedOutput, // either a revoked to_local output on commitment tx, a revoked HTLC-Timeout output or a revoked HTLC-Success output
+	RevokedOutput, // either a revoked to_holder output on commitment tx, a revoked HTLC-Timeout output or a revoked HTLC-Success output
```

Comment:
In the previous commit (while moving this block) you change it from to_holder to to_local, and then change it back here.

Reply:
Right, corrected in "Add onchain_utils"